src/main/java/nl/knaw/dans/easy/DataciteService.java

```diff
@@ -109,6 +92,27 @@ public boolean doiExists(String doi) throws DataciteServiceException {
         }
     }
 
+    private String postDoi(String content) throws DataciteServiceException {
```

Comment:
did anything change here? or is it just moving the method to a different place?

Reply:
Just moving. Sorting public and private methods

Reply:
👍